en/uavcan/README.md

```diff
@@ -21,6 +21,7 @@ It supports hardware like:
   * [Pomegranate Systems Power Module](../uavcan/pomegranate_systems_pm.md)
   * [CUAV CAN PMU Power Module](../uavcan/cuav_can_pmu.md)
 * Distance sensors
+  - [Ark Flow](../sensor/ark_flow.md)
```

Comment:
FYI added here. NOTE, this is new top level UAVCAN doc. You should consider linking to this doc for configuration and setup. The firmware/bootloader stuff is still in the development section.